.github/workflows/acquia_cms_cron.yml

```diff
@@ -384,8 +384,9 @@ jobs:
 
             # Remove drupal/core patch as it's needed for latest Drupal Core only.
             composer config extra.patches.drupal/core {} --json
-            sed -i 's/2598.patch",/2599.patch"/' modules/acquia_cms_common/composer.json
+            sed -i 's/2598.patch",/2598.patch"/' modules/acquia_cms_common/composer.json
             sed -i '/3356894-mr_3896.patch/d' modules/acquia_cms_common/composer.json
+            sed -i '/3370946-page-title-backport-10-2-2_0.patch/d' modules/acquia_cms_common/composer.json
```

Comment:
i think we can remove this change , once this gets merged or vice versa - https://github.com/acquia/acquia_cms/pull/1730/files

Reply:
The addition of above change necessary as it is not covered in the mentioned PR.